backend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "dahlia-1",
+  "version": "1.0.0",
+  "description": "parking app",
+  "main": "index.js",
+  "scripts": {
+    "test": "test_run"
+  },
```

Comment:
**suggestion:** A `start` or `dev` script to run the app would be nice.

`dev` script could use something like [`nodemon`](https://nodemon.io) for hot reload.

`start` script could just run `node app.js`.

Reply:
added both for app.js